pkg/webhooks/errors_test.go

```diff
@@ -66,3 +69,126 @@ func TestIsInvalidReplicationFactor(t *testing.T) {
 		t.Error("Expected is invalid replication error to be false, got true")
 	}
 }
+
+func TestIsCantConnectAPIServer(t *testing.T) {
+	testcases := []struct {
+		testname string
```

Comment:
nit - `test` and `name` are two words
```suggestion
		testName string
```

Reply:
I tried to respect the names already in use in some other file but I can't find where I looked.  
I'll update this in all the instances in the file.